link/src/main/java/com/meshconnect/link/entity/LinkConfiguration.kt

```diff
@@ -0,0 +1,7 @@
+package com.meshconnect.link.entity
+
+data class LinkConfiguration(
+    val token: String,
+    val accessTokens: List<IntegrationAccessToken>? = null,
+    val transferDestinationTokens: List<IntegrationAccessToken>? = null
+)
```

Comment:
to maintain consistency between platforms I suggest to follow the structure we implemented in iOS SDK
```suggestion
public struct LinkSettings {
    public var useSecureOnDeviceStorage: Bool
    public var accessTokens: [IntegrationAccessToken]?
    public var transferDestinationTokens: [IntegrationAccessToken]?
...
public class LinkConfiguration {
    var linkToken: String
    var settings: LinkSettings?```

Reply:
From an Arch perspective `LinkSettings` is not required.